certified-connectors/Elfsquad Data/apiDefinition.swagger.json

```diff
@@ -194,6 +194,27 @@
         "x-ms-visibility": "internal"
       }
     },
+    "/data/1/{entity_name}/$patch-schema": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@thomasio101 empty schema is not allowed. Please define a proper schema.

Reply:
@Amjed-Ayoub, I've resolved this for you :)